modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/AbstractKeyValueStorage.java

```diff
@@ -104,6 +124,13 @@ protected AbstractKeyValueStorage(String nodeName, FailureManager failureManager
     /** Returns key values by revision for operation. */
     protected abstract Value valueForOperation(byte[] key, long revision);
 
+    /**
+     * Returns {@code true} if the storage is in recovery state and the watches have not {@link #startWatches started}.
```

Comment:
This javadoc is confusing. For the Rocksdb storage, watches can be started, while we are still "in recovery". So I guess the javadoc should be:

```
Returns {@code true} if the storage is in the recovery state.
```